src/common/icons/cancel-icon.tsx

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+
+import { NamedSFC } from '../react/named-sfc';
+
+const d =
+    'M7.99121 7L14 13.0088L13.0088 14L7 7.99121L0.991211 14L0 13.0088L6.00879 7L0 0.991211L0.991211 0L7 6.00879L13.0088 0L14 ' +
+    '0.991211L7.99121 7Z';
+
+export const CancelIcon = NamedSFC('CancelIcon', () => (
+    <svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These svg elements should probably all have alt attributes. I'd expect the cancel icon should have "real" alt text (since it doesn't have a separate element acting as a descriptive label), and the other icons should probably have `alt=""` (since they are part of buttons with reasonable labels next to them already).

Since determining which case an icon is in relies on the context of how the icons are used, I think it would be more appropriate to add a prop to control this rather than embedding the decision in the individual icons.

Reply:
Added attributes according to the rules for images from Assessment